src/Jackalope/Version/VersionHandler.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Jackalope\Version;
+
+use Jackalope\Session;
+use Jackalope\Transport\AddNodeOperation;
+use Jackalope\Transport\WritingInterface;
+use PHPCR\InvalidItemStateException;
+use PHPCR\NodeInterface;
+use PHPCR\PropertyType;
+use PHPCR\UnsupportedRepositoryOperationException;
+use PHPCR\Util\UUIDHelper;
+use PHPCR\Version\VersionException;
+
+class VersionHandler
+{
+    const MIX_VERSIONABLE = 'mix:versionable';
+    const MIX_SIMPLE_VERSIONABLE = 'mix:simpleVersionable';
```

Comment:
this is a problem with the NodeTypeInterface constants using expanded namespaces. @lsmith77 any idea how we could solve that without too much overhead? duplicating constants here feels wrong to me.

maybe fix NodeType::isNodeType to translate between short and long namespace form and use that method instead of string comparison?